dropwizard/service/src/main/resources/application.properties

```diff
@@ -0,0 +1,158 @@
+quarkus.http.cors.access-control-allow-credentials=true
+
+quarkus.http.port=8181
+quarkus.http.test-port=0
+
+quarkus.log.level=INFO
+quarkus.log.console.enable=true
+quarkus.log.console.level=ALL
+quarkus.log.console.json=false
+quarkus.log.console.format=%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{3.}] [%X{requestId},%X{realmId}] [%X{traceId},%X{parentId},%X{spanId},%X{sampled}] (%t) %s%e%n
+quarkus.log.file.enable=true
+quarkus.log.file.level=ALL
+quarkus.log.file.json=false
+quarkus.log.file.format=%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{3.}] [%X{requestId},%X{realmId}] [%X{traceId},%X{parentId},%X{spanId},%X{sampled}] (%t) %s%e%n
+quarkus.log.file.path=./logs/polaris.log
+quarkus.log.file.rotation.file-suffix=.yyyy-MM-dd.gz
+quarkus.log.file.rotation.max-file-size=10M
+quarkus.log.file.rotation.max-backup-index=14
+quarkus.log.category."org.apache.polaris".level=INFO
+quarkus.log.category."org.apache.iceberg.rest".level=INFO
+quarkus.log.category."io.smallrye.config".level=INFO
+
+quarkus.management.enabled=true
+quarkus.management.port=8182
+quarkus.management.test-port=0
+
+quarkus.micrometer.enabled=true
+quarkus.micrometer.export.prometheus.enabled=true
+
+quarkus.otel.enabled=true
+quarkus.otel.sdk.disabled=false
+# quarkus.otel.exporter.otlp.endpoint=http://otlp-collector:4317
+# quarkus.otel.resource.attributes=deployment.environment=dev
+# quarkus.otel.service.name=polaris
+# quarkus.otel.traces.sampler=parentbased_always_on
+# quarkus.otel.traces.sampler.arg=1.0d
+
+polaris.context.realm-context-resolver.default-realm=default-realm
+polaris.context.realm-context-resolver.type=default
+
+polaris.config.defaults.ENFORCE_PRINCIPAL_CREDENTIAL_ROTATION_REQUIRED_CHECKING=false
+polaris.config.defaults.SUPPORTED_CATALOG_STORAGE_TYPES=["S3","GCS","AZURE","FILE"]
+# realm overrides
+# polaris.config.realm-overrides."my-realm".INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_TEST=true
+# polaris.config.realm-overrides."my-realm".SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION=true
+
+polaris.persistence.metastore-manager.type=in-memory
+
+polaris.io.file-io-factory.type=default
+
+polaris.log.request-id-header-name=request_id
+# polaris.log.mdc.aid=polaris
+# polaris.log.mdc.sid=polaris-service
+
+polaris.metrics.tags.application=Polaris
+# polaris.metrics.tags.service=polaris
+# polaris.metrics.tags.environment=prod
+# polaris.metrics.tags.region=us-west-2
+
+# polaris.otel.span-attributes.service=polaris
+# polaris.otel.span-attributes.environment=prod
+# polaris.otel.span-attributes.region=us-west-2
+
+# polaris.tasks.max-concurrent-tasks=100
+# polaris.tasks.max-queued-tasks=1000
+
+polaris.rate-limiter.type=default
+polaris.rate-limiter.token-bucket.type=default
+polaris.rate-limiter.token-bucket.requests-per-second=9999
+polaris.rate-limiter.token-bucket.window=PT10S
+
+polaris.authentication.type=default
+polaris.authentication.oauth2-service.type=default
+polaris.authentication.token-broker-factory.type=rsa-key-pair
```

Comment:
I included the "no-op" broker, but I would like to keep the new behavior as close as possible to the old one.

Reply:
But on `main` the default token broker is the "no-op" one, is it not?

Reply:
You are right, I forgot that there is this special handling in `PolarisApplicationConfig`:

https://github.com/apache/polaris/blob/1071aa24d2c2bad3ff0f9840a6b2572bc29e4e6a/dropwizard/service/src/main/java/org/apache/polaris/service/dropwizard/config/PolarisApplicationConfig.java#L242-L269

That said, in `main` the default authenticator is the "test" one, but I don't think we want to keep that one as the default authenticator going forward (or do we?).

Taking a step back, I think we want to 1) have good defaults for security and 2) have a default setup that starts up without errors.

The thig is, I'm not sure the "no-op" token broker with the "default" authenticator make a good pair; I'm getting the below error:

```
{"error":"unsupported_grant_type","error_description":"The grant type is invalid","error_uri":null}
```

I suggest a slightly better combination: "default" authenticator + "symmetric-key" token broker with a default secret of `changeme`. WDYT?

Another option, of course, would be to keep the "test" authenticator as the default one.

Reply:
yeah, tricky situation. IMHO using the "symmetric key" JWT with a well-known key by default is not much better. How about using the RSA with an auto-generated key?

This will allow single-node operation OOTB, if I'm not mistaken. Users who want to run distributed Polaris will have to reconfigure JWT signing keys.

Reply:
Good idea, I will do as you suggested. We can also print a warning to make it clear that this setup is only for evaluation purposes.